Encounter/encounter_role.md

```diff
@@ -0,0 +1,95 @@
+# Encounter Role
+
+## Overview
+
+* An Encounter role is specific to the [encounter](encounter.md). While these could match up to existing organizational roles (e.g., “Nurse”), 
+they don’t have to (e.g., “Lead Surgeon”).
+
+## Available operations. 
+
+1. [List encounter roles](#list-encounter-roles)
+2. [Create an encounter role](#create-a-encounter-role)
+3. [Update an encounter role](#update-a-encounters-role)
+4. [Delete an encounter role](#delete-a-encounters-role)
+
+
+### List encounter roles
+
+* #### List all non-voided encounter roles.
+    
+    Quickly filter encounter roles with given query parameters.Returns a `404 Not Found` status if encounter roles not exists. 
+     If user not logged in to perform this action,a `401 Unauthorized` status returned.
+    
+    ##### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *q* | `Search Query` | Query to filter encounter role by it's name (i.e., no apostrophe)
```

Comment:
LOL. Sorry, I had to chuckle at this little misunderstanding. 🙂 

When I said "no apostrophe", I didn't mean that this should literally be made part of the description; rather, I was trying to point out that "it's" (using an apostrophe) is a contraction for "it is". So, your description reads: "Query to filter encounter role by it is name", which is grammatically incorrect. I think what you intended was "Query to filter encounter role by its name" (note that "its" doesn't have an apostrophe). Sorry for not being clearer.